### PR TITLE
Connector/Intellisense: do not suggest x-ms-visibility:internal functions

### PR DESCRIPTION
**Pull request by LucGenetier**

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs [hunk 1]
```diff
@@ -55,6 +55,9 @@ internal sealed class ServiceFunction : BuiltinFunction, IAsyncTexlFunction2, IH
         // Check will return a warning when used
         private readonly bool _isDeprecated;
 
+        // For functions that have "x-ms-visibility" set to "internal" in swagger file.
+        private readonly bool _isInternal;
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs:59]: How is PowerApps handling this?

They must hit the same issue when importing Sharepoint swagger.

Comment (LucGenetier) [reply src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs:59]: They do

**Review by MikeStall — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**